Hardcore.lua

```diff
@@ -144,6 +144,57 @@ local ALERT_STYLES = {
 -- the big frame object for our addon
 local Hardcore = CreateFrame("Frame", "Hardcore", nil, "BackdropTemplate")
 Hardcore.ALERT_STYLES = ALERT_STYLES
+Hardcore.verify_shown = false
+
+
+--[[ Ace3 config and imports ]]--
+
+Hardcore.Ace3 = LibStub("AceAddon-3.0"):NewAddon("Hardcore", "AceEvent-3.0", "AceConsole-3.0")
+local AceGUI = LibStub("AceGUI-3.0")
```

Comment:
Can we move this to top of file?

Reply:
It can only be executed after the `Hardcore` Frame is created, on line 145:

If we wanted to move everything to the start of the file, we can do that, so long as we have everything we need already init'd before we create `Hardcore`